.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+{"version":1,"defects":{"dnj\\Ticket\\Test\\Fature\\DepartmentControllerTest::testShowList":4,"dnj\\Ticket\\Test\\Fature\\DepartmentControllerTest::testStore":4,"dnj\\Ticket\\Test\\Fature\\DepartmentControllerTest::testSearch":4,"dnj\\Ticket\\Test\\Fature\\DepartmentControllerTest::testShow":4,"dnj\\Ticket\\Test\\Fature\\DepartmentControllerTest::testUpdate":4,"dnj\\Ticket\\Test\\Fature\\DepartmentControllerTest::testDelete":4},"times":{"dnj\\Ticket\\Test\\Fature\\DepartmentControllerTest::testShowList":0.267,"dnj\\Ticket\\Test\\Fature\\DepartmentControllerTest::testStore":0.043,"dnj\\Ticket\\Test\\Fature\\DepartmentControllerTest::testSearch":0.051,"dnj\\Ticket\\Test\\Fature\\DepartmentControllerTest::testShow":0.035,"dnj\\Ticket\\Test\\Fature\\DepartmentControllerTest::testUpdate":0.047,"dnj\\Ticket\\Test\\Fature\\DepartmentControllerTest::testDelete":0.051}}
```

Comment:
This file should not commit to repo. Please remove it and put it's name in .gitignore